lib/seattleflu/id3c/cli/command/etl/redcap_det_asymptomatic_swab_n_send.py

```diff
@@ -23,7 +23,7 @@
 LOG = logging.getLogger(__name__)
 
 
-REVISION = 1
+REVISION = 2
```

Comment:
How do we decide when to bump a revision number?

Reply:
We bump revision number when we need to re-process all records in the receiving table